overlord/overlord_test.go

```diff
@@ -692,7 +721,13 @@ func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOpera
 }
 
 func (ovs *overlordSuite) TestEnsureLoopPruneAbortsOld(c *C) {
-	restoreIntv := overlord.MockPruneInterval(100*time.Millisecond, 24*time.Hour, 1*time.Hour)
+	restoreEnsureIntv := overlord.MockEnsureInterval(10 * time.Hour)
```

Comment:
Curious, why is this set? The default of 5min seems to be long enough to avoid this running during this particular test? Maybe a small comment if it's needed?

Reply:
This is to highlight the fact that this particular interval is not relevant anymore for this test; I'll add a comment to clarify.